Example/Messaging/App/iOS/FIRAppDelegate.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 Google
```

Comment:
`2018`? and below for new docs.

Reply:
Borderline - since I made an exact copy from an existing version with that copy. Justification in #1044